noir_stdlib/src/hash/mod.nr

```diff
@@ -105,7 +105,7 @@ fn __derive_generators<let N: u32, let M: u32>(
     // Same as from_field but:
     // does not assert the limbs are 128 bits
     // does not assert the decomposition does not overflow the EmbeddedCurveScalar
-    fn from_field_unsafe(scalar: Field) -> EmbeddedCurveScalar {
+    pub fn from_field_unsafe(scalar: Field) -> EmbeddedCurveScalar {
```

Comment:
```suggestion
fn from_field_unsafe(scalar: Field) -> EmbeddedCurveScalar {
```

This function is not intended to be public. However I think the other two are fine to be exposed.